docs/user/contextplugin.md

```diff
@@ -0,0 +1,27 @@
+# FlowForge Persistent Context Plugin
```

Comment:
I don't think the the user cares that it's a plugin? So can we change the filename to `persistent_context.md` and remove "Plugin" from this line?

Reply:
Done (next commit)